scene/gui/margin_container.cpp

```diff
@@ -80,6 +80,23 @@ Vector<int> MarginContainer::get_allowed_size_flags_vertical() const {
 	return flags;
 }
 
+int MarginContainer::get_margin_size(Side p_side) const {
+	ERR_FAIL_INDEX_V((int)p_side, 4, 0);
```

Comment:
I'm wondering if it wouldn't be better to introduce SIDE_MAX instead of hard-coding.
Then again, there are only 4 possible sides and it's not going to change 🤔

Reply:
Yeah, I had the same thoughts, and same conclusions. It's not going to change, and we're doing this in a few places already. Adding `MAX` may require to update a bunch of switches too just to please the compiler, so it's probably not worth it.